openpype/hosts/maya/plugins/publish/validate_node_ids_unique.py

```diff
@@ -29,8 +32,13 @@ def process(self, instance):
         # Ensure all nodes have a cbId
         invalid = self.get_invalid(instance)
         if invalid:
-            raise RuntimeError("Nodes found with non-unique "
-                               "asset IDs: {0}".format(invalid))
+            label = "Nodes found with non-unique asset IDs"
+            raise PublishValidationError(
+                message="{}: {}".format(label, invalid),
+                title="Non-unique asset ids on nodes",
+                description="{}\n- {}".format(label,
+                                              "\n- ".join(sorted(invalid)))
+            )
```

Comment:
This ain't pretty - how should we write this in a readable manner in code to get a decent report out in the new publisher?